content/features/proxy2/configure.md

```diff
@@ -0,0 +1,411 @@
+-- Define one default router handler, as a catch-all for any prefixes not matched
+-- in the map section, as well as keys with no prefixes at all
+default = {{<var>}}HANDLER_TYPE{{</var>}}{ {{<var>}}HANDLER_ARGS{{</var>}} },
+```
+
+Replace the following:
+
+* <var>MODE</var>: (optional) the technique that the proxy applies to determine the prefix of each key. The value must be one of the following:
+
+    * `prefix`: The prefix is every character from the start of the key until the characters defined by the separate `stop` argument. This is the default.
+    
+    * `anchor`: The prefix is every character between the two substrings defined by the separate `start` and `stop` arguments.
+    
+* <var>START_STRING</var>: if the mode is `anchor`, then this is a substring of five or fewer characters at the start of every key name that precedes the prefix.
+
+* <var>STOP_STRING</var>: a substring that separates the prefix from the remainder of the key name.
+
+* <var>KEY_PREFIX</var>: the key prefix to map to this route handler.
+    
+* <var>HANDLER_TYPE</var>: the name of a route library handler. For a list of all of the handlers available, see [Route handler reference]({{<proxy_base_path>}}reference).
+
+* <var>HANDLER_ARGS</var>: the arguments to pass to the route handler. Typically, this is where you name one or more pools. The list of arguments and their expected syntax depends on the handler type. For more information, see [Route handler reference]({{<proxy_base_path>}}reference).
+
+### Example configuration files
+
+The following example file demonstrates all four of the configuration file sections described earlier on this page:
+
+```lua
+-- Turn on the proxy's verbose mode.
+verbose(1)
+
+-- Override a handful of default settings for proxy behavior.
+settings {
+    active_request_limit = 100,
+    pool_options = {
+        filter = "tags",
+        filter_conf = "{}"
+    }
+}
+
+-- Define two backend pools of Memcached servers. Each pool has three nodes.
+pools {
+    main_pool = {
+        backend_options = { connecttimeout = 5 },
+        backends = {
+            "192.0.2.1",
+            "192.0.2.2",
+            "192.0.2.3:11212",
+        }
+    },
+    customer_pool = {
+        backends = {
+            "203.0.113.1",
+            "203.0.113.10",
+            { host = "203.0.113.20", port = 11212, connecttimeout = 10 },
+        },
+    },
+}
+
+-- Define two routes, each of which maps to one of the pools defined earliers.
+-- And then define a default catch-all route.
+routes {
+    map = {
+        -- Handle keys that start with "main/"
+        main = route_allfastest {
+            children = { "main_pool" },
+        },
+        cust = cmdmap{
+            -- Handle only SET commands for keys that start with "cust/"
+            set = route_allfastest {
+                children = { "customer_pool" },
+            },
+        },
+    },
+
+    -- Handle keys that don't have a mapped prefix.
+    default = route_allfastest {
```

Comment:
`route_direct`

Reply:
Fixed, and changed the argument to `child = {...}`.